onnxruntime/test/framework/ort_model_only_test.cc

```diff
@@ -308,6 +309,26 @@ TEST(OrtModelOnlyTests, SerializeToOrtFormatMLOps) {
 #endif  // #if !defined(DISABLE_ML_OPS)
 #endif  // #if !defined(ORT_MINIMAL_BUILD)
 
+TEST(OrtModelOnlyTests, SparseInitializerHandling) {
+  const std::basic_string<ORTCHAR_T> ort_file = ORT_TSTR("sparse_initializer_handling.onnx.ort");
```

Comment:
This need to be moved into #if !defined(ORT_MINIMAL_BUILD)
We cannot convert the model in minimal build